content/docs/rfcs/22/README.md

```diff
@@ -0,0 +1,46 @@
+---
+slug: 22
+title: 22/CHAT2
+name: Waku v2 Chat
+status: raw
```

Comment:
Maybe add category: application even if we don't index it now?

Reply:
I hesitated to do it now or do it for both this and DM-ETH in a follow-up PR. Will do now then.